Containerfile

```diff
@@ -50,23 +50,45 @@ RUN cargo build \
     --bin shuttle-next -F next
 
 
-# The final image for running each "shuttle-..." binary
+# Base image for running each "shuttle-..." binary
 ARG RUSTUP_TOOLCHAIN
-FROM docker.io/library/rust:${RUSTUP_TOOLCHAIN}-buster as shuttle-crate
+FROM docker.io/library/rust:${RUSTUP_TOOLCHAIN}-buster as shuttle-crate-base
 ARG CARGO_PROFILE
 ARG folder
 ARG crate
 ARG prepare_args
-# used as env variable in prepare script
+# Fixes some dependencies compiled with incompatible versions of rustc
+ENV RUSTUP_TOOLCHAIN=${RUSTUP_TOOLCHAIN}
```

Comment:
We need this to be set in the deployer containers, and I don't think this will do that (at least it doesn't look like it on staging currently).

Reply:
Ah, I missed that the ARG before FROM is cleared after the FROM.